vanilla-framework/vanilla-framework
# 2100 spacing variable rename

## Done

Renamed spacing variables based on various requests for simplification and memorability.

The changes mostly follow these rules:
- Intra/Inter -> Inner/Outer
- Condensed -> Small / x-small (depends how it's used and if there are multiple levels)
- Expanded -> Large
- Regular -> Medium
- (unmodified) -> Medium

I've kept the `--scaleable` modifier, which is chained with the other sizes.

One exception to the S/M/L rule is the vertical outer spacing which uses shallow/deep to keep in line with the p-strip (and elsewhere) terminology

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- See that everything is the same

## Details

Fixes #2100 
Fixes #2140 
Fixes #1923
